src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from 'react';
+import { DispatchContext, TodosContext } from '../../Store';
+
+export const Footer: React.FC = () => {
+  const {
+    todos,
+    selectedAll,
+    selectedActive,
+    selectedCompleted,
+  } = useContext(TodosContext);
+  const dispatch = useContext(DispatchContext);
+
+  const todosNotCompleted = [...todos].filter(todo => todo.completed === false);
+  const numberNotCompletedTodos = todosNotCompleted.length;
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${numberNotCompletedTodos} items left`}
+      </span>
+
+      <ul className="filters">
+        <li>
+          <a
+            href="#/"
+            className={`${selectedAll && 'selected'}`}
+            onClick={() => dispatch({ type: 'selectedAll' })}
```

Comment:
It would be better to prevent the default behavior of the click event to avoid changing the URL when clicking on the link. You can do this by passing an event to the onClick function and calling event.preventDefault().